levenshtein.go

```diff
@@ -0,0 +1,44 @@
+package emailverifier
+
+// levenshteinDistance calculate the distance between two string
+// Refer to the implementation of https://github.com/hbollon/go-edlib/blob/master/levenshtein.go
```

Comment:
why not use that library directly?

Reply:
Because only the levenshtein algorithm is used so far, it looks simple enough not to import the package. But now I found that it is easier to import the package directly！